website/content/02-guides/03-server.mdx

```diff
@@ -3,32 +3,33 @@ title: 'Server'
 metaTitle: 'Server'
```

Comment:
Why even use `Server` when it is just about `Serverless`? cc @jasonkuhrt

Reply:
Or is this the component name?

Reply:
Heya,

So Nexus has a "server" component that can be either in serverful or serverless mode.

For example:

```ts
import { server } from 'nexus'

server.middleware() // ... this would work over serverless OR serverful
```

From a product POV this is still a WIP, we'll see how #782 goes + evolves...

To your question, the guides, it is not that the guide title was intended to match the component name. However, we felt that having two guides `serverless` and `serverful` would be confusing. Also a lot of content that applies to one applies to the other (e.g. middleware).

We wanted a single guide for all "server related stuff", and concluded on the name server. Within it, there will be three parts:

1. serverless only stuff
2. serverful only stuff
3. general stuff shared by all modes

Now that you have the context, do you have a suggestion for a better guide name?

Reply:
I would write both Schema component and Server component:

![image](https://user-images.githubusercontent.com/849648/85674920-8c7c7280-b6c5-11ea-8d43-d21f5ebcd23f.png)

Further down the nav they are referred to as components:

![image](https://user-images.githubusercontent.com/849648/85675122-ba61b700-b6c5-11ea-965e-3e529036a560.png)

(is 'logging' also a 'logger component')?